scripts/dap-ami-bootstrap.sh

```diff
@@ -1,38 +1,42 @@
 #!/bin/bash
 set -e
 
-if ! docker images | grep conjur; then
-  echo "Loading Conjur appliance image into Docker"
-  gzip -df /home/core/conjur-appliance.tar.gz # Unzip the tar.gz into a tar
-  docker load -i /home/core/conjur-appliance.tar
+amazon-linux-extras install -y docker
+systemctl enable docker
+systemctl start docker
+
+if ! docker images | grep appliance; then
+  echo "Loading DAP appliance image into Docker"
+  docker load -i /opt/dap-appliance.tar.gz
 fi
 
 image_id=$(docker images -q)
-container_name='conjur-appliance'
+container_name='dap-appliance'
 
 docker rm -f ${container_name} > /dev/null 2>&1 || true
 
 mkdir -p /var/log/conjur
 mkdir -p /opt/conjur/backup
 
-echo "Creating Conjur container"
+echo "Creating DAP container"
 cid=$(docker create \
 --name ${container_name} \
 --privileged --restart always \
```

Comment:
Can we change this to:
```
--restart=unless-stopped \
```
This prevents a down container from coming up if it's been stopped and the instance is restarted.